src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -597,6 +597,13 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Corax.Static.ComplexFieldIndexingBehavior", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public CoraxComplexFieldIndexingBehavior CoraxStaticIndexComplexFieldIndexingBehavior { get; protected set; }
 
+        [Description("Interval of saving elapsed time from last query in an idle index")]
+        [DefaultValue(10)]
+        [TimeUnit(TimeUnit.Minutes)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.IntervalOfStoringElapsedTimeFromLastQueryInMin", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
```

Comment:
Typically we use `...IntervalInMin` suffix in configuration names so maybe `Indexing.ElapsedSinceLastQueryPersistIntervalInMin`

Reply:
done